packages/utils/src/carequality/cq-monthly-stats/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { QueryTypes } from "sequelize";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import fs from "fs";
+import { sleep } from "@metriport/shared";
+import { xcpdStats } from "./xcpd-stats";
+import { xcaDQStats } from "./xca-dq-stats";
+import { xcaDRStats } from "./xca-dr-stats";
+import { readOnlyDBPool } from "./shared";
+
+dayjs.extend(duration);
+
+async function main() {
+  const sqlCQDirectory = `SELECT * FROM cq_directory_entry`;
```

Comment:
Does this script do well with the default 2GB mem? The cq directory is quite hefty AFAIK. Just curious. If not, we can add a comment to run this with teminal prefix: 
`node --max-old-space-size=8192 -r`